lib/perl/Genome/Model/Tools/Sx/Trim/Flexbar.pm

```diff
@@ -281,6 +282,18 @@ sub _resolve_adapters {
     return 1;
 }
 
+sub _resolve_length_dist_parameter {
+    my $self = shift;
+
+    my $no_length_dist = $self->no_length_dist;
+
+    if ( $self->version >= 230 ) {
```

Comment:
I've moved this one up by the others.

I made mine a `>=` because there is a newer release of `flexbar` available, and it also follows the newer parameter convention. I've changed the others where appropriate.  (The environment variable looks version specific.)

On another note, the two installed versions are `/usr/bin/flexbar` and `/gsc/bin/flexbar`.  The public versions are all 2.x whereas we have integer version numbers.  Currently `230` is `2.4` and `229` is `2.21`.  This seems suspicious.